src/components/Link/Link.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
We should integrate this functionality with RouterLinkWrapper which was setup in prep for routing injection.

Reply:
oh snap, I'll look into that

Reply:
Granted RouterLinkWrapper doesn't leap out as 'Link' substitute :)